packages/analytics/src/analytics.ts

```diff
@@ -0,0 +1,65 @@
+import * as analytics from 'universal-ga';
+import { ILTAnalytics, ILTAnalyticsOptions } from 'lt-analytics';
+
+export class LTAnalytics implements ILTAnalytics {
+  private allTrackersIDs: string[];
+  private options: ILTAnalyticsOptions;
+
+  constructor(options: ILTAnalyticsOptions) {
+    this.options = options;
+    this.allTrackersIDs = this.options.trackers;
+
+    if (!this.allTrackersIDs || this.allTrackersIDs.length < 1) {
+      throw new Error('Configuration Error. You must specify at least one tracker');
+    }
+
+    const defaultTrackingID = this.allTrackersIDs[0];
+    analytics.initialize(defaultTrackingID, options.createOptions);
```

Comment:
I think you are creating the same tracker twice. Once in `analytics.initialize` and again in `this.allTrackersIDs.forEach`